test/swivel/Sig.sol

```diff
@@ -12,7 +12,8 @@ library Sig {
 
   error S();
   error V();
-  error SIG_LENGTH();
```

Comment:
no other errors are `CONST_STYLE()` so remoing in favor of `Length`. The library same is `Sig` so i think context lets us know its the signature length